src/aspire/operators/wemd.py

```diff
@@ -0,0 +1,63 @@
+"""
+Wavelet-based approximate Earthmover's distance (EMD) for n-dimensional signals.
+
+This code is based on the following paper:
+    Sameer Shirdhonkar and David W. Jacobs.
+    "Approximate earth mover’s distance in linear time."
+    2008 IEEE Conference on Computer Vision and Pattern Recognition (CVPR).
+
+More details are available in their technical report:
+    CAR-TR-1025 CS-TR-4908 UMIACS-TR-2008-06.
+"""
+
+import numpy as np
+import pywt
+
+
+def wemd_embed(arr, wavelet, level):
```

Comment:
Sensible defaults for `wavelet` and `level` (same for `wemd_norm` below).

Reply:
I'm not sure I want to set default parameters:
1. Taking the level to be log2 is just something I found experimentally to give smooth results--and using larger levels doesn't seem to do much except take more space and runtime--so I've added a comment in the docstring about it.
2. Regarding the choice of wavelets: different wavelets seem to work better in different contexts. I've seen good results with coif3 and sym5, but don't have enough experience to provide solid recommendations at this time.

Reply:
So why not make these things default? They're not hardcoded, just a place to start. If someone wants to override them, they're free to. The point is that you're in a better position to choose these values than the average user will be, so let's take advantage of that.

Does the paper have any recommendations for these parameters? What do they use in their experiments?

Reply:
Ok.
In the paper they have different theoretical guarantees for the various wavelets, which may or may not be meaningful and they've also tested a bunch of them. They say they got good results with coif3 and sym5 and picked coif3 for their simulations. I also got good results with these two in experiments, so I picked "coif3" as the default.